Assets/Scripts/Game/UserInterfaceWindows/DaggerfallUnitySaveGameWindow.cs

```diff
@@ -462,6 +470,18 @@ protected virtual void SetBackground(Button button, Color color, string textureN
 
         #endregion
 
+        #region Private Methods
+        private void LoadLatestSaveGameIntoSaveNameTextBox()
+        {
+            int mostRecentSave = GameManager.Instance.SaveLoadManager.FindMostRecentSave();
+            SaveInfo_v1 latestSaveInfo = GameManager.Instance.SaveLoadManager.GetSaveInfo(mostRecentSave);
+            currentPlayerName = latestSaveInfo.characterName;
+            saveNameTextBox.Text = latestSaveInfo.saveName;
+            screenshotPanel.BackgroundTexture = new Texture2D(0, 0);
```

Comment:
Couldn't we fill this with the save's picture, or do we want to avoid loading a file without waiting for user action?

Reply:
Hey, I wrote that (line 480) only to have the flux work as usual without breaking.